### ISSUE TITLE
Running script fails when passing null to a function

### ISSUE THREAD

**Issue by shanethehat** [jenkinsci/JenkinsPipelineUnit#284]
In the following example I am getting a failure at the point of executing the Jenkinsfile:

Jenkinsfile:
```
@Library('my-library') _

pipeline {
  agent none
  stages {
    stage('Foo') {
      steps {
        script {
          def foo = doSomething(null)
        }
      }
    }
  }
}
```

doSomething.groovy:
```
def call(String foo) {
  if (null != foo) {
    echo foo
  }
}
```

DoSomethingTest:
```
...

class DoSomethingTest extends DeclarativePipelineTest {
    @Override
    @BeforeAll
    void setUp() throws Exception {
        scriptRoots += 'vars'

        super.setUp()

        // generic pipeline mocking
        helper.registerAllowedMethod('pipeline', [Closure.class], null)
        helper.registerAllowedMethod('agent', [Closure.class], null)
        helper.registerAllowedMethod('stages', [Closure.class], null)
        helper.registerAllowedMethod('steps', [Closure.class], null)
        binding.setVariable('none', {})

        // library registration
        def library = library()
                .name('my-library')
                .retriever(projectSource())
                .targetPath("<notNeeded>")
                .defaultVersion("<notNeeded>")
                .allowOverride(true)
                .implicit(true)
                .build()
        helper.registerSharedLibrary(library)
    }

    @Test
    void testDoSomething() {
        runScript("job/library/Jenkinsfile")
        printCallStack()
        assertJobStatusSuccess()
    }
}
```

This results in a stacktrace that originates from the runScript command of the test and starts
> DoSomethingTest > testDoSomething() FAILED
    java.lang.IllegalArgumentException: wrong number of arguments

If I replace the `null` in the Jenkinsfile with `""` then all is well.

---

Sample repo: https://github.com/shanethehat/sample-null-pipeline-library

**Comment by shanethehat** [comment]
I just tried to fix this and found that it's already fixed in master by commit 58b1bf9. Is there any chance for this getting a release soon?

**Comment by stchar** [comment]
Hi @shanethehat , sure will release it soon.
